### DISCUSSION TITLE
Combine Unloading Mode Question

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2340] [General]
Not sure if this is a courseplay question or autodrive tbh. When unloading a combine why does the tractor keep trying to follow like the forager? In real life this would only happen when the combine reches about 80% full then it would follow to unload then stop and wait again. If the trailer didn't get to the combine in time, the combine would wait. Seems rather odd to have a tractor and trailer following a combine which has a bunker around the field all the time or is there a setting for this I'm missing somewhere?

**Comment by Tensuko** [comment]
Is CP unloading the combine or AD?

**Reply by coatsy35** [reply]
From the autodrive HUD, I have unload combine mode and deliver to store. I just wasn't sure which bit controls the machine in the field as the combine is running courseplay cutting and the trailer keeps running by the side of it like a forager.

**Reply by Tensuko** [reply]
If you have set nothing for CP unloader then it is AD unloading the combine and then you have to ask pon AD side as I have no Idea about their unload combine mode.

**Reply by rohne83** [reply]
That's your Autodrive settings then. There's a setting for active unloading. If you deactivate that the unloader should stop unloading as soon as the combine is empty.
